src/main/scala/nl/knaw/dans/easy/license/Command.scala

```diff
@@ -15,16 +15,92 @@
  */
 package nl.knaw.dans.easy.license
 
-import org.slf4j.LoggerFactory
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, FileOutputStream, OutputStream}
+
 import nl.knaw.dans.easy.license.{CommandLineOptions => cmd}
+import org.slf4j.LoggerFactory
+import rx.lang.scala.{Observable, ObservableExtensions}
+import rx.lang.scala.schedulers.IOScheduler
+import rx.schedulers.Schedulers
+
+import scala.collection.JavaConverters._
+import scala.language.postfixOps
+
+class Command(datasetLoader: DatasetLoader,
+              placeholderMapper: PlaceholderMapper,
+              templateResolver: TemplateResolver,
+              pdfGenerator: PdfGenerator)(implicit parameters: Parameters) {
```

Comment:
I'm not happy with the name Command. Might be convenient to spot as main class in this project. However, the pattern applied so far for projects called by `easy-ingest-flow` is `EasyIngest.run(ingest.Settings(...`, `EasyStageDataset.run(stage.Settings(` etc. Please follow the same pattern.